2200039/10515.py

```diff
@@ -0,0 +1,8 @@
+T = int(input())
+for i in range(T):
+    population = int(input())
+    income = list(map(int,input().split()))
+    average = sum(income) // population; count = 0
```

Comment:
```suggestion
    average = sum(income) / population; count = 0
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄